DataFormats/L1TMuon/interface/EMTF/GEM.h

```diff
@@ -0,0 +1,86 @@
+// Class for Gas-Electron Multiplier (GEM) EMTF Data Record
+
+#ifndef __l1t_emtf_GEM_h__
+#define __l1t_emtf_GEM_h__
+
+#include <vector>
+#include <cstdint>
+
+namespace l1t {
+  namespace emtf {
+    class GEM {
+    public:
+      explicit GEM(const uint64_t dataword);
+
+      GEM()
+          : pad(-99),
+            partition(-99),
+            cluster_size(-99),
+            cluster_id(-99),
+            link(-99),
+            gem_bxn(-99),
+            bc0(-99),
+            tbin(-99),
+            vp(-99),
+            format_errors(0),
+            dataword(-99){};
+
+      virtual ~GEM() = default;
+
+      inline void set_pad(const int bits) { pad = bits; }
```

Comment:
please use the naming convention http://cms-sw.github.io/cms_coding_rules.html#2----naming-rules-1 for member variables, getters, and setters in this class.

Reply:
This class was added based on the template of other classes in the `interface/EMTF` under the assumption that local continuity had higher importance than global style, not as an endorsement of any particular style choice.
Such an uniformization throughout the remainder of the code was left as a potential followup PR